src/legacy/ui/public/capabilities/react/inject_ui_capabilities.tsx

```diff
@@ -18,7 +18,7 @@
  */
 
 import React, { Component, ComponentClass, ComponentType } from 'react';
-import { UICapabilities } from '../ui_capabilities';
+import { UICapabilities } from '../../../../../core/public/capabilities';
```

Comment:
**optional** could you import `Capabilities` from `../../../../../core/public`? we want to restrict imports from root internals. right now linter is in progress https://github.com/elastic/kibana/pull/34688